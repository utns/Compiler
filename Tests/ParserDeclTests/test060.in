procedure foo(a: integer b: integer);
begin
end;