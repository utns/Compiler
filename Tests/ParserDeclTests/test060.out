Syntax error. Expected ')' but 'b' found: line = 1 pos = 26
